tests/acceptance/test_artificial.py

```diff
@@ -26,7 +26,7 @@ class TestVunitArtificial(unittest.TestCase):
 
     def setUp(self):
         # Spaces in path intentional to verify that it is supported
-        self.output_path = join(dirname(__file__), "artificial out")
+        self.output_path = join(dirname(__file__), "artificial_out")
```

Comment:
The comment in the line above makes me think that it was a concious decision to use a space instead of an underscore.
Why not simply vunit/test/acceptance/artificial\ out in .gitignore?